reference/functions/packagesmatching.markdown

```diff
@@ -39,7 +39,13 @@ this:
 The following code extracts just the package names, then looks for
 some desired packages, and finally reports if they are installed.
 
-**IMPORTANT:** Please note that you need to provide `package_inventory` attribute in `body common control` in order to be able to use this function. Also depending on the value(s) of `package_inventory` only packages from selected package modules will be returned. For more information about `package_inventory` please read [`package_inventory`][Components#package_inventory] section.
+**IMPORTANT:** Querying operates on either standard package modules or legacy package methods.
+In order to query standard package modules you must provide a list of modules you want to query in a `package_inventory` attribute in `body common control`.
```

Comment:
It's strange to me to say querying "package modules".

Maybe .../

When package_inventory in body common control is configured cfengine will record the packages installed and the package updates available for the configured package modules.